docs/api/general_rest_usage.md

```diff
@@ -208,6 +208,17 @@ The REST API supports two authentication methods:
 
 Session-based is the default authentication method, as it is needed for UI.
 
+!!! note
+    
+    There are some API endpoints that might be accessible for the Anonymous User outside the Back Office due to missing permissions or limitations. To prevent that, you can rely on the Symfony securing URL patterns mechanizm called [access_control](https://symfony.com/doc/3.4/security/access_control.html). The example below shows how to disallow listing Content Types for the not-authenticated user.
```

Comment:
Look good 👍  Only things I see to potentially improve would be to consider to add a header, and maybe tweak the first sentence a bit.

```suggestion
!!! note "Anonymous access to meta data over REST"
    
    Some API endpoints accessible for the Anonymous User return metadata you might not want exposed, due to their nature in PHP API they don't have own permission limitations. To prevent that, you can rely on the Symfony securing URL patterns mechanizm called [access_control](https://symfony.com/doc/3.4/security/access_control.html). The example below shows how to disallow listing Content Types for the not-authenticated user.
```

Reply:
From my perspective it looks good, I will let @ezsystems/documentation-team decide how to put this.